functorch/_src/partitioners.py

```diff
@@ -368,7 +380,10 @@ def get_node_weight(node):
         if ban_recomputation(node) and node in required_fw_nodes:
             nx_graph.add_edge("source", node.name + "_in", capacity=math.inf)
 
-        if 'tensor_meta' not in node.meta:
+        if is_sym_node(node):
+            weight = 1
+        # TODO(whc) try changing this to isTensor(node.meta['val'])?
```

Comment:
I think the last time I read the code, the current state is that:

`node.meta['val']` gets set to the `FakeTensor`, or `SymIntNode` for that fx noe
`node.meta['tensor_meta']` gets set to a `TensorMeta` object, that's just a dataclass that's meant to hold a bunch of tensor metadata


So `node.meta['val']` is probably more accurate to use (the `tensor_meta` field doesn't properly track `requires_grad` for example, AND it won't track symints), but you can only use it when fake tensors are turned on. And... fake tensors are turned on by default in our symbolic shapes branch, right? But I guess they aren't turned on everywhere yet.